C/tests/c4QueryTest.cc

```diff
@@ -911,7 +911,7 @@ N_WAY_TEST_CASE_METHOD(NestedQueryTest, "C4Query Nested UNNEST", "[Query][C]") {
             "Univ of Pennsylvania, student_112, 3, piano",  "Univ of Pennsylvania, student_112, 3, swimming",
             "Univ of Pennsylvania, student_189, 2, violin", "Univ of Pennsylvania, student_189, 2, movies"};
 
-    for ( int withIndex = 1; withIndex <= 1; ++withIndex ) {
+    for ( int withIndex = 0; withIndex <= 1; ++withIndex ) {
```

Comment:
Consider using Catch `GENERATE` macro

Reply:
This style is apparently dominant for the batch of tests of this nature. It loops twice, first time w/o index, the second time w/ the index. It's also natural because it applies to the same db, same query, and expecting same result.